website/blog/2021-06-03-experience-share-from-Apache-APISIX-committer.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "来自 Apache APISIX committer 的经验分享 —— 编程之夏专访"
+author: Serendipity96
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+keywords:
+- committer
```

Comment:
??

Reply:
What is your suggestion？

Reply:
🤔 I don't think we need this as keyword..

Reply:
I edited the keywords